dotnet/docs/EXPERIMENTS.md

```diff
@@ -6,65 +6,67 @@ You can use the following diagnostic IDs to ignore warnings or errors for a part
 
 ```xml
 <PropertyGroup>
-  <NoWarn>SKEXP0001,SKEXP0011</NoWarn>
+  <NoWarn>SKEXP0001,SKEXP0010</NoWarn>
 </PropertyGroup>
 ```
 
-## Core
-
-- SKEXP0001: Embedding services
-- SKEXP0002: Image services
-- SKEXP0003: Memory connectors
-- SKEXP0004: Kernel Filters
-- SKEXP0005: Audio services
-
-## OpenAI and Azure OpenAI services
-
-- SKEXP0010: Azure OpenAI with your data service
-- SKEXP0011: OpenAI embedding service
-- SKEXP0012: OpenAI image service
-- SKEXP0013: OpenAI parameters
-- SKEXP0014: OpenAI chat history extension
-- SKEXP0015: OpenAI file service
-
-## Memory connectors
-
-- SKEXP0020: Hugging Face AI connector
-- SKEXP0021: Azure AI Search memory connector
-- SKEXP0022: Chroma memory connector
-- SKEXP0023: DuckDB memory connector
-- SKEXP0024: Kusto memory connector
-- SKEXP0025: Milvus memory connector
-- SKEXP0026: Qdrant memory connector
-- SKEXP0027: Redis memory connector
-- SKEXP0028: Sqlite memory connector
-- SKEXP0029: Weaviate memory connector
-- SKEXP0030: MongoDB memory connector
-- SKEXP0031: Pinecone memory connector
-- SKEXP0032: Postgres memory connector
-
-## Functions
-
-- SKEXP0040: GRPC functions
-- SKEXP0041: Markdown functions
-- SKEXP0042: OpenAPI functions
-- SKEXP0043: OpenAPI function extensions
-
-## Out-of-the-box plugins
-
-- SKEXP0050: Core plugins
-- SKEXP0051: Document plugins
-- SKEXP0052: Memory plugins
-- SKEXP0053: Microsoft 365 plugins
-- SKEXP0054: Web plugins
-- SKEXP0055: Text chunkcer plugin
-
-## Planners
-
-- SKEXP0060: Handlebars planner
-- SKEXP0061: OpenAI Stepwise planner
-
-## Experiments
-
-- SKEXP0101: Experiment with Assistants
-- SKEXP0102: Experiment with Flow Orchestration
+## Experimental Feature Codes
+
+| SKEXP​ | Experimental Features Category​​ |
+|-------|--------------------------------|
+| SKEXP0001 | Semantic Kernel core |
+| SKEXP0010 | OpenAI and Azure OpenAI services |
+| SKEXP0020 | Memory connectors |
+| SKEXP0040 | Function types |
+| SKEXP0050 | Out-of-the-box plugins |
+| SKEXP0060 | Planners |
+| SKEXP0100 | Experiments |
+
+## Experimental Features Tracking
+
+| SKEXP​ | Features​​ | API docs​​ | Learn docs​​ | Samples​​ | Issues​​ | Implementations​ |
+|-------|----------|----------|------------|---------|--------|-----------------|
+| SKEXP0001 | Embedding services | | | | | |
+| SKEXP0001 | Image services | | | | | |
+| SKEXP0001 | Memory connectors | | | | | |
```

Comment:
Isn't already the `Connectors` category for it? Maybe update to Memory Services Abstractions?
```suggestion
| SKEXP0001 | Memory services | | | | | |
```